cpdb_api/request.py

```diff
@@ -32,6 +35,22 @@ def set_request(self, r):
         """
         self._request = r
 
+    def set_api_user(self, u):
```

Comment:
Just leave it as a note here: we should ask Leo what should we do for authentication in production server.